### ISSUE TITLE
Progresso/frontend/relatório.

### ISSUE THREAD

**Issue by matheusgr** [lucasduarte2/DACA---projeto-web-para-farmacias#2]
Algum progresso aqui?

**Comment by lucasduarte2** [comment]
Sim, só que não subi pro git, fiz uma página que cadastra, mostra e com
opção de deletar medicamentos vou subir ainda hj pro git, e ainda tem uma
atualização no backend que também ainda não subi

Em 03/08/2018 19:46, "Matheus Gaudencio do Rêgo" <notifications@github.com>
escreveu:

**Comment by lucasduarte2** [comment]
Matheus poderia da uma olhada no BackEnd ? ainda vou arrumar a api rest de funcionário
